src/main/java/com/googleinterns/zoomtube/data/Feedback.java

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.data;
+
+import com.google.appengine.api.datastore.Key;
+import com.google.auto.value.AutoValue;
+
+/** Contains data related to feedback left by clicking icons. */
+@AutoValue
+public abstract class Feedback {
```

Comment:
Did we want to name this something like `IconFeedback` to distinguish it from potentially other types of feedback?